docs/contributing.md

```diff
@@ -65,6 +65,10 @@ Not everything is enforced or validated by the schema. A few things to pay atten
 
 If the feature you're interested in is a JavaScript API, you can cross-reference data against [Web API Confluence](https://web-confluence.appspot.com/) using the `confluence` command. This command overwrites data in your current working tree according to data from the dashboard. See [Using Confluence](using-confluence.md) for instructions.
 
+## Optional: Generating data using the mirroring script
+
+Many browsers within BCD are derived from Chrome. To help cut down time working on copying these values over, a mirroring script is provided. You can run `npm run mirror <browser> <feature_or_file> [--source=""] [--modify=""]` to automatically perform this mirroring process. The browser is the destination browser that values will be copied to. The script automatically determines what browser to copy from based upon the destination, but manual specification is possible through the `--source=""` argument. The feature argument is the identifier of the feature to update (i.e. `css.at-rules.namespace`), a filename (`javascript/operators/arithmetic.json`), or an entire folder (`api`). By default, the mirroring script will only overwrite values in the destination that are `true` or `null`, but can take a `--modify=""` argument to specify whether to overwrite values that are `false` as well (`--modify=bool`), or any values (`--modify=always`).
```

Comment:
Should we maybe expand a bit how automatic mirroring works? "Many browsers within BCD are derived from Chrome" makes this sound like the mirroring script only takes care about Chromium derivatives, but it also mirrors say Firefox Android from Firefox, right? So, maybe a little table that shows what is mirrored from where automatically would help a little?